<launch>
<!-- Set Parameters-->
    <param name="rs_video" type="int" value="1" />
    <param name="rs_depth" type="int" value="1" />

<!-- Launch Nodes-->
    
    <node name="senor_manager" pkg="lattepanda_behaviours" type="sensor_manager.py" output="screen"/>  
    <node name="behaviour_ros" pkg="lattepanda_behaviours" type="behaviour_main.py" output="screen"/>  
    <node name="realsense_behaviour" pkg="lattepanda_behaviours" type="realsense.py" />
    <node name="navigator_behaviour" pkg="lattepanda_behaviours" type="navigator.py" />
    <node name="force_behaviour" pkg="lattepanda_behaviours" type="force.py" output="screen"/> 
    <node name="light_behaviour" pkg="lattepanda_behaviours" type="light.py" output="screen"/> 
    <node name="microphone_behaviour" pkg="lattepanda_behaviours" type="microphone.py" output="screen"/> 

    <node name="plot_vectors" pkg="lattepanda_behaviours" type="plot_vectors.py" />    

</launch>